testing/test_recwarn.py

```diff
@@ -477,3 +477,14 @@ def test_catch_warning_within_raise(self) -> None:
             with pytest.raises(ValueError, match="some exception"):
                 warnings.warn("some warning", category=FutureWarning)
                 raise ValueError("some exception")
+
+    def test_multiple_arg_custom_warning(self) -> None:
```

Comment:
Let's just add a docstring to mention the original issue for future reference (I know we can eventually find it via git-blame, but often things move around, code is refactored/reformatted, so I think it pays to just slap the issue number along with the test):

```suggestion
    def test_multiple_arg_custom_warning(self) -> None:
        """pytest.warns with custom warning subclass with multiple arguments (#11906)."""
```